fighters/miigunner/src/opff.rs

```diff
@@ -134,7 +134,7 @@ unsafe fn arm_rocket_airdash(boma: &mut BattleObjectModuleAccessor, id: usize, s
 unsafe fn lunar_launch_actionability(fighter: &mut L2CFighterCommon, boma: &mut BattleObjectModuleAccessor, motion_kind: u64, frame: f32) {
     if [hash40("special_hi1"),
         hash40("special_air_hi1")].contains(&motion_kind) {
-        if frame >= 35.0 && VarModule::get_float(boma.object(), vars::miigunner::status::CHARGE_ATTACK_LEVEL) <= 10.0 {
+        if boma.status_frame() >= 35 && VarModule::get_float(boma.object(), vars::miigunner::status::CHARGE_ATTACK_LEVEL) <= 10.0 {
```

Comment:
Was this done to fix a bug? Generally, if you’re checking status kind, you should be using status frames, and if you’re checking motion kind, you should be using motion frames.